lib/manageiq/cross_repo/test_core.rb

```diff
@@ -0,0 +1,30 @@
+module ManageIQ::CrossRepo
+  class TestCore < TestBase
+    attr_reader :core_repo, :gem_repos
+
+    def initialize(core_repo, gem_repos)
+      raise "You must pass at least one gem to override" if gem_repos.to_a.empty?
```

Comment:
Lets raise an ArgumentError instead of RuntimeError.

Reply:
:+1: done